geoportal/c2cgeoportal_geoportal/lib/authentication.py

```diff
@@ -44,14 +47,17 @@ def create_authentication(settings):
     http_only = http_only.lower() in ("true", "yes", "1")
     secure = settings.get("authtkt_secure", "True")
     secure = secure.lower() in ("true", "yes", "1")
-    if len(settings["authtkt_secret"]) < 64:
+    samesite = settings.get("authtkt_samesite", "Lax")
+    secret = settings["authtkt_secret"]
+    if len(secret) < 64:
         raise Exception('"authtkt_secret should be at least 64 characters.'
                         'See https://docs.pylonsproject.org/projects/pyramid/en/latest/api/session.html')
 
     cookie_authentication_policy = AuthTktAuthenticationPolicy(
-        settings["authtkt_secret"],
+        secret,
         callback=defaultgroupsfinder,
         cookie_name=settings["authtkt_cookie_name"],
+        samesite=samesite is None if samesite == '' else samesite,
```

Comment:
samesite == '' => samesite is None == False
So it could be simplified to:
```
False if samesite == '' else samesite
```

Reply:
Oups thanks :-)

Reply:
=> https://github.com/camptocamp/c2cgeoportal/pull/5127

Reply:
But False is not a correct value for samesite: 
```
samesite

    Default: 'Lax'. The 'samesite' option of the session cookie. Set the value to None to turn off the samesite option.
```
So it should be correct and really simpler with:
```
samesite=samesite or None
```

Reply:
Note: for boolean settings it would be better to use pyramid.settings.asbool